enhancements/network/ovn-kubernetes-multiple-external-gateway.md

```diff
@@ -0,0 +1,180 @@
+---
+title: ovn-kubernetes-multiple-external-gateway
+authors:
+  - "@jordigilh"
+reviewers: 
+  - "@trozet"
+  - "@tssurya"
+  - "@fedepaol"
+approvers:
+  - "@trozet"
+api-approvers: 
+  - "@trozet"
+creation-date: 2023-02-06
+last-updated: 2023-02-17
+tracking-link: 
+  - https://issues.redhat.com/browse/SDN-2481
+---
+
+# OVN-Kubernetes Multiple External Gateways
+
+## Summary
+
+When it comes to ingress traffic, load balancers and reverse proxies take care of redirecting traffic to one of potentially multiple destinations. Egress traffic is no different. Although it is already possible to use egress gateways, telco customers require the capability to use a common and secure API to dynamically configure one or more egress next hops per namespace.
+
+Multiple External Gateway is an existing OVN-K feature that uses external gateways for ingress and egress traffic for all the pods in an annotated namespace, using the following pod and namespace. More information can be found in this [white paper](https://www.f5.com/pdf/white-papers/f5-telco-gateway-for-red-hat-openshift-wp.pdf).
+The list of annotations used in the current implementation are as follow:
+
+* Namespace
+  * `k8s.ovn.org/routing-external-gws`
+  * `k8s.ovn.org/bfd-enabled`
```

Comment:
pending discussion from https://github.com/openshift/enhancements/pull/1338#discussion_r1111883787 ; based on that we might need to update the info on `external-gw-pod-ips`